test/cmocka/Makefile.am

```diff
@@ -49,6 +49,12 @@ check_PROGRAMS += alloc
 alloc_SOURCES = src/lib/alloc/alloc.c src/lib/alloc/mock.c ../../src/lib/alloc.c ../../src/platform/apollolake/memory.c
 endif
 
+# lib/lib tests
+
+check_PROGRAMS += rstrlen
+rstrlen_SOURCES = src/lib/lib/rstrlen.c
+rstrlen_LDADD = ../../src/lib/libcore_a-lib.o $(LDADD)
```

Comment:
I think we shouldn't link to object (.o) files, better to link to lib that you test (.a or .la) file

Reply:
Fixed as per request.